eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -240,7 +240,7 @@ stages:
     dependsOn: Signing
     jobs:
     - job: PublishPackages
-      condition: or(eq(variables['SetDevVersion'], 'true'), and(eq(variables['Build.Reason'],'Schedule'), eq(variables['System.TeamProject'], 'internal')))
+      condition: or(eq(variables['SetDevVersion'], 'true'), and(in(variables['Build.Reason'],'Schedule', 'IndividualCI'), eq(variables['System.TeamProject'], 'internal')))
```

Comment:
We shouldn't do this here in fact we should probably consider removing anything other then the SetDevVersion condition here. For example if this is a CI run and it doesn't have the dev version set then this process will fail after the first publish because the package versions will not be unique.

Reply:
Yeah good point, if SetDevVersion is true then we can always run Integration.